.prettierrc.yaml

```diff
@@ -5,4 +5,8 @@ semi: true
 singleQuote: false
 trailingComma: "es5"
 bracketSpacing: true
-arrowParens: "avoid"
\ No newline at end of file
+arrowParens: "avoid"
+overrides:
+  - files: "*.mdx"
+    options:
+      printWidth: 60
```

Comment:
This is to just auto format mdx files so our mintlify code blocks stay narrow enough to be displayed in the sidepanel w/out scroll

Reply:
Awesome! 🤩